drivers/sensor/bma400/bma400.c

```diff
@@ -0,0 +1,339 @@
+/*
+ * Copyright (c) 2020 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT bosch_bma400
+
+#include <drivers/i2c.h>
+#include <init.h>
+#include <drivers/sensor.h>
+#include <sys/__assert.h>
+#include <logging/log.h>
+
+#include "bma400.h"
+
+LOG_MODULE_REGISTER(bma400, CONFIG_SENSOR_LOG_LEVEL);
+
+static int reg_read(u8_t reg, u8_t *data, u16_t length,
+	struct bma400_dev *dev);
+static int reg_write(u8_t reg, const u8_t *data, u16_t length,
+	struct bma400_dev *dev);
+static int bma400_sample_fetch(struct device *dev, enum sensor_channel chan);
+static void channel_accel_convert(struct sensor_value *val, s64_t raw_val,
+	u8_t range);
+static int set_accel_config(const struct bma400_acc_cfg *cfg,
+	struct bma400_dev *dev);
+static s16_t convert_acc_data(u8_t lsb, u8_t msb);
+static int set_power_mode(u8_t power_mode, struct bma400_dev *dev);
+static int set_attributes(double odr, double osr, double range,
+	struct bma400_dev *dev);
+
+int bma400_init(struct device *dev)
```

Comment:
Typically init is on the bottom.

Reply:
@jfischer-phytec-iot , This is the preferred style and would prefer keeping it that way.

Reply:
It is not, the most of the driver have init on the bottom. Please fix this style issues.